include/rpt/rpt_utilities.h

```diff
@@ -51,7 +51,8 @@ attach_grid_to_triangulation(Triangulation<dim> &             triangulation,
                                      5,
                                      parameters.reactor_radius,
                                      parameters.reactor_height / 2.);
-  GridTools::rotate(M_PI_2, 1, triangulation);
+  Tensor<1, dim> axis({0, 1, 0});
```

Comment:
I don't understand why here you defined axis (0,1,0) while the other files you defined it as axis(0,0,0) and the changed axis[rotation_axis] to 1. What is the difference between the two cases?

Reply:
In this case this rotation is hardcoded to be in this directoin. In the other one it's with a user paraemter. This is a code we need to make more general in the future :)